README.md

```diff
@@ -11,7 +11,18 @@ aware of.
 - Tabzilla is no longer available as a hosted service. You have to add it to your
 website using one of the options below.
 - Tabzilla no longer needs any fonts bundled with it so, we are no longer including
-`@font-face` declarations for Open Sans in the CSS.
+`@font-face` declarations for Open Sans in the CSS. *NOTE:* You can still load
+the Open-Sans fonts using the Mozilla CDN, or another CDN of your choosing, for example:
+
+```CSS
+@font-face{
+	font-family:'Open Sans Light';
+	src:url(//mozorg.cdn.mozilla.net/media/fonts/OpenSans-LightItalic-webfont.eot);
```

Comment:
Any reason not to always use `https://` here? Also, maybe `.woff` is a better example?